docs/data/material/pages.ts

```diff
@@ -236,6 +236,11 @@ const pages = [
       { pathname: '/material-ui/discover-more/languages' },
     ],
   },
+  {
+    pathname: 'https://mui.com/store/?utm_source=docs&utm_medium=referral&utm_campaign=sidenav',
+    title: 'Templates',
+    icon: 'ReaderIcon',
```

Comment:
```suggestion
    icon: 'StoreRoundedIcon',
```

Reply:
Would it look like this?

<img width="88" alt="Screenshot 2022-05-06 at 14 51 01" src="https://user-images.githubusercontent.com/3165635/167134779-d0bd81e2-5806-497b-a3fa-6e21a5f67813.png">
 
I think that it should communicate the idea of a template.